app/views/personal_access_tokens/new.html.erb

```diff
@@ -0,0 +1,9 @@
+<% title _('Create Personal Access Token for %s') % @user.name %>
+
+<ol class="breadcrumb">
```

Comment:
@ohadlevy: What do you think of something like this?
https://github.com/weppos/breadcrumbs_on_rails